docs/maintain-guides-how-to-vote-councillor.md

```diff
@@ -12,10 +12,15 @@ in the elections.
 
 ## Voting for Councillors
 
-Voting for councillors requires you to lock your DOT / KSM for the duration of your vote. Like the
-validator elections, you can approve up to 16 different councillors and your vote will be equalized
-among the chosen group. Unlike validator elections, there is no unbonding period for your reserved
-tokens. Once you remove your vote, your tokens will be liquid again.
+Voting for councillors requires you to lock 5 DOT on Polkadot or 0.0083 KSM on Kusama for the
```

Comment:
Just out of curiosity, where can I find the calculations behind this in the codebase?

Reply:
Constants -> ElectionsPhragmen -> VotingBond